org.eclipse.wb.swt/src/org/eclipse/wb/internal/swt/model/property/editor/color/ColorPropertyEditor.java

```diff
@@ -176,10 +177,9 @@ private String getText(Property property) throws Exception {
           Object keyArgument = invocation.arguments().get(0);
           if (keyArgument instanceof QualifiedName) {
             GenericProperty genericProperty = (GenericProperty) property;
-            ResourceRegistryInfo registry =
-                RegistryContainerInfo.getRegistry(
-                    genericProperty.getJavaInfo().getRootJava(),
-                    invocation.getExpression());
+            ResourceRegistryInfo registry = RegistryContainerInfo.getRegistry(

```

Comment:
Format changes and real changes should not be mixed. I like this reformat, please provide separate PR for this.